xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h

```diff
@@ -249,6 +254,17 @@ extern "C" {
     PVR_CONNECTION_STATE_CONNECTING         = 7,  /*!< @brief connecting to backend */
   } PVR_CONNECTION_STATE;
 
+   /*!
+   * @brief PVR recorded TV item state types
+   */
+  typedef enum
+  {
+    PVR_RECORDED_TV_ITEM_STATE_IN_PROGRESS          = 0,  /*!< @brief recording is in progress */
+    PVR_RECORDED_TV_ITEM_STATE_ERROR                = 1,  /*!< @brief recording not started because of error */
+    PVR_RECORDED_TV_ITEM_STATE_FORCED_TO_COMPLETION = 2,  /*!< @brief recording was forced to completion, but may miss a certain part at the end because it was cancelled by user */
+    PVR_RECORDED_TV_ITEM_STATE_COMPLETED            = 3,  /*!< @brief recording completed successfully */
+  } PVR_RECORDED_TV_ITEM_STATE;
```

Comment:
I would go with just "PVR_RECORDING_STATE".

Reply:
Enums are bad for several reasons. Please define int constants for the possible states.

Reply:
I just copied these from pvr.dvblink but that is better.